index.js

```diff
@@ -20,47 +21,68 @@ async function createRelease(repoName,tag, name, body) {
 }
 
 try {
+
+
+  // check if draft has been released from master
+  const payload = JSON.stringify(github.context.payload, undefined, 2)
+  const release = !!(payload.action && payload.action === 'released');
+  if (release) {
+    // build the package and test it
+    execSync(`yarn`);
+    execSync(`yarn build`);
+    execSync(`yarn test`);
+    console.log('....Publishing Package....');
+    execSync(`npm publish`);
```

Comment:
@karkipy 12 barsa ramayan padhayo staging ra production laie different banaie halyo

Reply:
> @karkipy Why is there two index.js file ?

one if using a compiled version of index.js in root, dist/index.js solves the issue of having node modules

Reply:
action would most likely be using only one index.js file and not both, so figure out which one it is and then keep that one only. It will be a nightmare to maintain two index.js file and then forgetting to update the one that is required and double checking every time.